### ISSUE TITLE
maxgames collab?

### ISSUE THREAD

**Issue by maxwick456** [v4for/v4for.github.io#4]
Hey, i noticed that josh removed you from the credits of strongdog, its happened to me too, and i just happened to be looking for someone to help with css because i suck at it. my links are https://maxgames.pages.dev https://maxj.me https://99v.org https://unblockedgamespremium.pages.dev and others. I have 50k in the last 7 days.
![image](https://github.com/user-attachments/assets/bb6b1cb8-427f-413f-9432-8a5cebc9cc3a)
lmk if you are interested. I am @mjfw on discord

**Comment by jman1593** [comment]
no max, his name i still in the credits, the reason it was gone for a while is cuz i did a ton of changes in the index.html and forgot to add his name back (i removed credits for some testing)

also, i removed YOURS because i removed all your code because you absolutely obliterated my trust.

stop trying to turn all my friends (or friends older brothers) against me because of a mistake YOU made

**Comment by maxwick456** [comment]
to be fair, you have turned a lot of people on me

**Comment by jman1593** [comment]
who? Also, you have and are doing the same to me.

**Comment by maxwick456** [comment]
ig your right, i have no one to turn on me

**Comment by jman1593** [comment]
no, you do, i know at least 1 person and a few others i have suspicions about, but its not like im going to even try cuz im not trying to ruin your life or get rid of your friends.
